bokeh/util/tests/test_image.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import absolute_import
+
+import unittest
+import numpy as np
+
+from bokeh.util.image import convert_gray_to_bokehrbga, convert_rgb_to_bokehrbga
+
+class TestConvertRgbToBokehRgba(unittest.TestCase):
+    def test_basic(self):
+        img = np.ones((10, 100, 3), dtype=np.uint8)
```

Comment:
This is nitpicky, but the tests doesn't use the unittest assert methods. This makes the error messages less readable. I'd refactor to use the unittest assert methods or (because we just added pytest compatibility), write pytest-style tests.